recipe/meta.yaml

```
@@ -1,76 +1,82 @@
   host:
     - python
     - poetry-core >=1.0.0
     - pip
   run:
     - python
-    # as referenced in https://github.com/pydantic/pydantic/issues/9637#issuecomment-2162873917, older than 2.7.4
-    # presents an error with py312 "TypeError: ForwardRef._evaluate() missing 1 required keyword-only argument: 'recursive_guard'"
-    - pydantic >=2.5.2,<3.0.0  # [py<312]
-    - pydantic >=2.7.4,<3.0.0  # [py>=312]
     - langsmith >=0.1.125,<0.2.0
     - tenacity >=8.1.0,!=8.4.0,<10.0.0
     - jsonpatch >=1.33.0,<2.0.0
     - pyyaml >=5.3
-    - packaging >=23.2.0,<25.0.0
-    - typing-extensions >=4.7,<5.0
+    - packaging >=23.2,<25
+    - typing-extensions >=4.7
+    # https://github.com/langchain-ai/langchain/blob/langchain-core==0.3.19/libs/core/pyproject.toml#L36-L42
+    - pydantic >=2.5.2,<3.0.0  # [py<312]
+    - pydantic >=2.7.4,<3.0.0  # [py>=312]
+
+{% set tests_to_ignore = "" %}
+# Ignore because missing syrupy:
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/prompts/test_chat.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/prompts/test_prompt.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/runnables/test_fallbacks.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/runnables/test_graph.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/runnables/test_runnable.py" %}
+# Ignore because missing langchain_tests:
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/indexing/test_in_memory_indexer.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/stores/test_in_memory.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=unit_tests/vectorstores/test_in_memory.py" %}
+{% set tests_to_skip = "" %}
+# Skip test_as_import_path because tarball paths don't match expected
+{% set tests_to_skip = tests_to_skip + "test_as_import_path" %}
+# Skip flaky test_benchmark_model, test_same_event_loop because time based tests
+{% set tests_to_skip = tests_to_skip + " or test_benchmark_model" %}
+{% set tests_to_skip = tests_to_skip + " or test_same_event_loop" %}
+# Skip test_rate_limit_invoke on win because time/tick dependent
+{% set tests_to_skip = tests_to_skip + " or test_rate_limit_invoke" %}  # [win]
+# Skip the following on win because use InMemoryDocumentIndex in beta and it fails
+# I have not found much information about it. For other platforms it works.
+{% set tests_to_skip = tests_to_skip + " or test_cleanup_with_different_batchsize" %}  # [win]
+{% set tests_to_skip = tests_to_skip + " or test_index_into_document_index" %}  # [win]
 
 test:
-  source_files:
-    - libs/core
   imports:
     - langchain_core
+  source_files:
+    - libs/core/tests/unit_tests
+    - libs/core/tests/__init__.py
   requires:
     - pip
-    - pytest >=7.3.0,<8
-    - pytest-mock >=3.10.0,<4  # [not (osx and x86 and py>=312)]
-    - numpy >=1.24.0,<2
-    - responses >=0.25.0,<0.26.0
+    - pytest >=7.3.0
```

```suggestion
    - pytest >=8.0.0,<9.0.0
```
https://github.com/langchain-ai/langchain/blob/langchain-core%3D%3D0.3.19/libs/core/pyproject.toml#L105

---

`nothing provides requested pytest >=8.0.0,<9.0.0`
so revert the previous pin` >=7.3.0`. I think it's fine too

---

reverted